language_detector/main.py

```diff
@@ -1,8 +1,9 @@
-# -*- coding: utf-8 -*-
-
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    counts = { lang['name']:0 for lang in languages }
+    words = text.split()
+    for w in words:
+        for lang in languages:
+            if w in lang['common_words']:
+                counts[lang['name']]+=1
+    return max((count, name) for name, count in counts.items())[1]
```

Comment:
thanks for your review, @StefGou! from looking at other groups' codes I realised we should've done `.lower()` on the input as well.

good point about the `return` statement - thanks!